docs/configuration.md

```diff
@@ -265,15 +299,30 @@ Restart each test runner worker process after `n` runs. Not recommended unless y
 ### `mutate` [`string[]`]
 
 Default: `['{src,lib}/**/*.js?(x)', '!{src,lib}/**/__tests__/**/*.js?(x)', '!{src,lib}/**/?(*.)+(spec|test).js?(x)', '!{src,lib}/**/*+(Spec|Test).js?(x)']`<br />
-Command line: `[--mutate|-m] src/**/*.js,a.js`<br />
-Config file: `"mutate": ["src/**/*.js", "a.js"]`
+Command line: `[--mutate|-m] "src/**/*.js","a.js"` (You need to use quotes on the command line , if you use patterns with `*` or `**` in order to prevent the shell from expanding them before stryker can read them, [see also](#configuration-options-and-globbing-expressions))<br />
```

Comment:
Actually the duplication was intentional, because sometimes people tend to read the documentation not from the top of the page and i thought it's important enough to repeat it here. but i will shorten it a bit and refer to the information at the top